kak-lsp.toml

```diff
@@ -99,7 +99,7 @@ command = "dls"
 filetypes = ["dart"]
 roots = ["pubspec.yaml", ".git", ".hg"]
 command = "sh"
-args = ["-c", "dart $(dirname $(command -v dart))/snapshots/analysis_server.dart.snapshot --lsp"]
+args = ["-c", "exec dart $(dirname $(command -v dart))/snapshots/analysis_server.dart.snapshot --lsp"]
```

Comment:
thanks - I don't think it's necessary for dart, the shell already uses exec semantics here